app/js/populateCategoryDropdown.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * populates the category dropdowns for the table
+ */
+function populateCategoryDropdown () {
+    getData('category').then((testsObject) => {
+        getTemplateAjax('js/templates/categoryFilter.hbs').then((HBTemplate) => {
```

Comment:
this feels like it should be split down into two further functions, one for each dropdown. incase you need to repopulate one without the other at some point

Reply:
done and changed the name of both